plugins/toolbox/src/components/Root/tools.tsx

```diff
@@ -230,6 +231,13 @@ export const defaultTools: Tool[] = [
     description: 'Calculate hash from given text',
     aliases: ['md2', 'md5', 'sha1', 'sha256', 'sha512'],
   },
+  {
+    id: 'ts-interface-generate',
+    name: 'JSON to Interface',
+    component: <Interface />,
+    category: 'Generate',
+    description: 'Instantly generate TypeScript interfaces from JSON',
```

Comment:
Maybe not just TS interfaces? Maybe something like `Generate interfaces for different programming languages from JSON`?

Reply:
Good idea, updated the PR